evap/static/ts/src/datagrid.ts

```diff
@@ -13,6 +14,7 @@ interface Row {
 interface State {
     search: string;
     filter: Map<string, string[]>;
+    rangedFilter: Map<string, Range>;
```

Comment:
What do you think about "range filter" instead of "ranged filter"? Seems more natural to me.